profiles/repositories.py

```diff
@@ -29,6 +39,21 @@ def next_id(self) -> str:
         raise NotImplementedError
 
 
+class SQLAlchemyOrcidTokens(OrcidTokens):
+    def __init__(self, db: SQLAlchemy) -> None:
+        self.db = db
```

Comment:
why not injecting only `self.db.session`? It has a too short lifetime or will `self.db` soon be needed?

Reply:
Just following code seen elsewhere. Reading the docs it looks safe to inject `db.sessions`.